app/src/main/java/com/keylesspalace/tusky/adapter/NotificationsAdapter.java

```diff
@@ -112,6 +113,7 @@ public NotificationsAdapter(String accountId,
     @Override
     public RecyclerView.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
         LayoutInflater inflater = LayoutInflater.from(parent.getContext());
+        absoluteTimeFormatter = new AbsoluteTimeFormatter(DateFormat.is24HourFormat(parent.getContext()));
```

Comment:
Pass a context into the NotificationsAdapter constructor then you can only create one AbsoluteTimeFormatter instead of one for every ViewHolder.

Reply:
The other ViewHolder does not build in `NotificationsAdapter`.
I think `AbsoluteTimeFormatter` is a UI Util tool, so I use ViewHolder context.

Reply:
How does that make a difference if it is only used to look up the 24h format?